src/decorations.ts

```diff
@@ -1,86 +1,41 @@
-import { window, OverviewRulerLane, DecorationRangeBehavior } from 'vscode';
+import {
+  window,
+  OverviewRulerLane,
+  DecorationRangeBehavior,
+  ExtensionContext,
+  TextEditorDecorationType,
+} from 'vscode';
 
-export function failingItName() {
+function createTestStateDecoration(
+  context: ExtensionContext,
+  icon: string,
+  overviewRulerColor: string
+): TextEditorDecorationType {
   return window.createTextEditorDecorationType({
-    overviewRulerColor: 'red',
+    overviewRulerColor,
+    gutterIconPath: context.asAbsolutePath(`images/icons/${icon}.svg`),
```

Comment:
will this path be a problem in windows? how about this

```suggestion
    gutterIconPath: context.asAbsolutePath(path.join('images', 'icons', `${icon}.svg`)),
```

Reply:
tested with on Win10+VSCodium, clean install. It does this internally I think

Reply:
changed to what is already in repo (see [src/Coverage/Formatters/GutterFormatter/index.ts:46](/jest-community/vscode-jest/blob/d84d24f9f2f39db43e327bf896e706e43ac4f10e/src/Coverage/Formatters/GutterFormatter/index.ts#L46))

Reply:
actually there is a bug regarding GutterFormatter icon (#585), but it might be a packaging issue and not necessarily the path... I don't have any windows machine, not sure if there is any variation that could cause a path problem... unless we are absolutely sure otherwise it is probably better to be safe... 

I actually think we should update GutterFormatter 1. the path formation, 2. consolidate the svg files to the same location as yours.